common/sgx/tlsverifier.c

```diff
@@ -63,7 +58,7 @@ oe_result_t verify_report_user_data(
 oe_result_t oe_verify_tls_cert(
```

Comment:
Chengmean, this API can be called from within an enclave and outside an enclave, correct? In both cases does it have to be compiled with SGX=ON?

Reply:
Thanks for pointing out this concern, which will be addressed when the following PR is merged.

https://github.com/microsoft/openenclave/pull/1575